src/Caching/StackExchangeRedis/src/PublicAPI/net8.0/PublicAPI.Shipped.txt

```diff
@@ -0,0 +1,26 @@
+#nullable enable
+Microsoft.Extensions.Caching.StackExchangeRedis.RedisCache
```

Comment:
This feels wrong, the new APIs should be the `unshipped.txt` files until we ship 9.0 RTM

Reply:
fixed; agreed this is "unshipped"